### ISSUE TITLE
Unable to turn off Direwolves

### ISSUE THREAD

**Issue by NielsPilgaard** [PrinceOfAmber/ERZ#9]
Heya.

I set `B:allBiomes=false` for Direwolves, and left the whitelist blank, however they can still spawn.

Minecraft 1.12.2
Forge 1.12.2-14.23.1.2566
EmberRootZoo-1.12-1.3.3

**Comment by Lothrazar** [comment]
Hey sorry i took so long to respond.  I investigated this a few weeks ago but forgot to come back and respond so ill write it up here.

So i tried to replicate this myself.  First i made a new world with a new config file and saw them spawn in a super flat world.

I set all biomes is false in 'entitydirewolf' , (they had no default biome whitelist.)

Then in a super flat world i used /kill @e to kill everything and let them respawn a bunch.  After I set to false i dont see any spawn.  I fly all around all over, kill everything to reset and try again.  Beofre when it was true i saw them all the time.

I will investigate more but so far this seems okay to me.
